metadata/Cadastre/land_ownership

```diff
@@ -0,0 +1,89 @@
+
+Standard colors are recommended when displaying this feature class. A layer file containing the standard color scheme is available by contacting SITLA GIS staff directly at the GIS staff from SITLA at [TLA-GIS@utah.gov] | 801-538-5100, or visiting [https://trustlands.utah.gov].
+
+<!--- Did some digging on the SITLA website and did not find the standard color scheme mentioned, so I believe users will need to contact SITLA directly if they are interested in using it. -->
+
+### What is the purpose of the dataset?
+
+Land ownership data is used by many government and private entities to demarcate boundaries, provide evidence in legal disputes, and display administrative jurisdictions for agencies such as the Bureau of Land Management and the US Forest Service. This dataset is also a helpful resource for public recreation, news outlets, and other interested parties.
+
+<!--- While this is general information on how land ownership data could be used, SITLA does have a disclaimer on their website that specifies that their data is not suitable for "legal, engineering, or surveying purposes" and that the data is for "reference only". Is this something we should include here as well?
+
+Here's the link if anyone wants to read their disclaimer: https://trustlands.utah.gov/tools/maps/ 
+t this point, I am unsure if this disclaimer applies to all their GIS data, or if just to a specific reference map. -->
+
+### What does the dataset represent?
+
+Land ownership types are divided into four categories: Federal, State, Tribal, and Private ownership. This dataset also includes official land use designation, the administrator of the parcel, and the area of the parcel in square meters. For a full explanation of the field names and descriptions, please see the [schema summary](https://docs.google.com/document/d/1CW9Y8MgxBEo53Hzs7ZdKeAfFeG_6cGwN4WOzAIBDreU/edit?usp=sharing).
+
+### How was the dataset created?
+
+<!--- Took a look around SITLA's website and found this layer, but it turns out our metadata is the same as theirs, so there isn't too much information about this layer there. Thus, I will be reaching out to SITLA to fill in some of the gaps in the metadata. Feel free to add notes about how this dataset was created if you have them! -->
+
+### How reliable and accurate is the dataset?
+
+Maintenance of this data layer is performed by a cooperative federal and state effort between the Bureau of Land Management (BLM) and the State of Utah School and Institutional Trust Lands Administration (SITLA). Both SITLA and the BLM update this dataset regularly.
+
+## Credits
+
+### Data Source
+
+- STILA
+- BLM
+- Local Partners
+
+### Host
+
+UGRC
+
+## Restrictions
+
+For more information on usage limitations for this dataset, please see our [disclaimer](https://gis.utah.gov/documentation/policy/license/#disclaimer).
```

Comment:
Should this go to sitlas website or use their language?

Reply:
I think both would be good. I'll include an abbreviated version of their disclaimer and that add a link for people to learn more.